helm/resource_release.go

```diff
@@ -617,13 +617,21 @@ func resourceDiff(d *schema.ResourceDiff, meta interface{}) error {
 		return nil
 	}
 
+	// Validates the resource configuration, the values, the chart itself, and
+	// the combination of both.
+	//
+	// Maybe here is not the most canonical place to include a validation
```

Comment:
I agree this is not the canonical place to put this. It would be better to put this in a `ValidateFunc` on the `chart` attribute, but there we don't have access to the provider meta struct so we wouldn't be able to validate with the supplied values.

Reply:
Yep, this was the only method I was able to make it work in the plan stage.